### PR TITLE
Helm support for clustered zookeeper

### PR DESCRIPTION
**Pull request by dgrove-oss**
Allow zookeeper.replicaCount to be set to values > 1 by
properly configuring the zookeeper cluster and adjusting other
parts of the Helm chart so that dependent services can
properly communicate with the cluster.

Also document which of the deployments do not yet support
multiple replicas in docs/configurationChoices.md and
via comments in values.yaml.

### CODE REVIEW

File: helm/templates/zookeeper.yaml [hunk 1]
```diff
@@ -20,14 +21,35 @@ spec:
...
+  name: {{ .Values.zookeeper.name | quote }}
+  namespace: {{ .Release.Namespace | quote }}
+data:
+  zoo.cfg: |
+    tickTime=2000
+    clientPort={{ .Values.zookeeper.port }}
+    initLimit=5
+    syncLimit=2
+    dataDir=/data
+    dataLogDir=/datalog
```
Comment (daisy-ycguo) [inline helm/templates/zookeeper.yaml]: I wonder if these values can be moved to values.yaml. If they can, it's good to have all configuration variables in a single place.

Comment (dgrove-oss) [reply helm/templates/zookeeper.yaml]: Good point -- thanks!  Done.

File: helm/values.yaml [hunk 2]
```diff
@@ -166,6 +170,8 @@ invoker:
 apigw:
   name: "apigateway"
   image: "openwhisk/apigateway:latest"
+  # sett
```
[the remark this answers is not in the document]
Comment (dgrove-oss) [reply helm/values.yaml]: fixed.

**Comment by dgrove-oss** [comment]
Addressed review comments, rebased, & squashed.  Should be ready to merge as soon as Travis is happy.

**Pull request merged**